docs/fragmentation.html

```diff
@@ -0,0 +1,411 @@
+	</p>
+	<p>	
+	We find that the model predicts fragmentation to be a major source of plastic particles in terms of abundance. On the other hand, fragmentation seems to play a minor role in 
+	the mass budget compared to other environmental sinks, by breaking down particles such that they become smaller than typical microplastics (<0.1mm). 
+    </p><p style="color:#9E9E9E">
+    	<small>author: <a href="https://www.uu.nl/medewerkers/MLAKaandorp">Mikael Kaandorp</a>, date: April 2021</small>
+    </p>
+    <a href="#abstract_expanded" class="collapsed btn btn-primary" data-toggle="collapse" aria-expanded="true">Read the scientific abstract</a>
+	<a class="collapsed btn btn-primary" href="https://iopscience.iop.org/article/10.1088/1748-9326/abe9ea"  target="_blank">Go to the article in Environmental Research Letters</a>
+	
+  <div id="abstract_expanded" class="collapse" role="tabpanel">
+    <div class="card-body">
+      	<p>
+		Field studies in the global ocean have shown that plastic fragments make up the majority of plastic pollution in terms of abundance. 
+		It is not well understood how quickly plastics in the marine environmental fragment, however. Here, we study the fragmentation process in the 
+		oceanic environment by considering a model which captures continuous fragmentation of particles over time in a cascading fashion. 
+		With this cascading fragmentation model we simulate particle size distributions (PSDs), specifying the abundance or mass of particles for different 
+		size classes. 
+		</p>
+		<p>	
+		The fragmentation model is coupled to an environmental box model, simulating the distributions of plastic particles in the ocean, coastal waters, and on the beach. 
+		We demonstrate the capabilities of the model by calibrating it to estimated plastic transport in the Mediterranean Sea, and compare the modelled PSDs to available observations in this region. 
+		Results are used to illustrate the effect of size-selective processes such as vertical mixing in the water column and resuspension of particles from the beach into coastal waters. 
+		</p>
+		<p>			
+		The model quantifies the role of fragmentation on the marine plastic mass budget: while fragmentation is a major source of secondary plastic particles in terms of abundance, it seems to have a minor effect on the total mass of particles larger than 0.1 mm. 
+		Future comparison to observed PSD data allow us to understand size-selective plastic transport in the environment, and potentially inform us on plastic longevity.     	
+		</p>
+    </div>
+  </div>
+
+  <hr>
+
+  <h2 id="introduction">Introduction: why should we care about plastic fragmentation?</h2>
+  <div class="row">
+    <div class="col-lg-7" style="margin-bottom: 20px">
+      <img class="media-object" src="images/fragmentation/overview.png" width=100% alt="">
+      <img class="media-object" src="images/fragmentation/Fragments_2.jpeg" width=90% alt="">
```

Comment:
This figure needs a reference and/or source information where you obtained it from?